nav-app/src/app/app-routing.module.ts

```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { HelpPageComponent } from './help-page/help-page.component';
+import { TabsComponent } from './tabs/tabs.component';
+
+const routes: Routes = [
+  { path: '', component: TabsComponent },
+  { path: 'help', component: HelpPageComponent }
```

Comment:
While this does work, I'm curious as to what it achieves? It doesn't seem like the `/help` route is linked to from anywhere else in the app. It does allow one to link directly to `/help` on the internet, but you can already link to the help markdown file on GitHub for the same effect (aforementioned bugs with that markdown notwithstanding).

Reply:
Or was my assumption that the markdown was intended to be consumed in GitHub itself incorrect, and the intention was to link to `/help` from the beginning? 

That would explain why it was so buggy when viewed in Markdown. But since it lives in the root directory of the repo I expect people would look at it there. We may want to move it somewhere within `src` (e.g the `assets` folder) if it's not supposed to be consumed directly on GitHub.

Reply:
Consuming it in Markdown on GitHub was how [the issue originally described it](https://github.com/mitre-attack/attack-navigator/issues/179#issue-643814179), but we can definitely change that if it's not easy to achieve due to the aforementioned image issues.